telemetry/definitions/commonDefinitions.json

```diff
@@ -1038,6 +1067,16 @@
             "type": "boolean",
             "description": "Whether or not the deploy targets a new destination (true) or an existing destination (false)"
         },
+        {
+            "name": "inputPolicyType",
+            "type": "string",
+            "allowedValues": [
+                "Identity",
```

Comment:
same as above comment. do we want to include "Other"

Reply:
Maybe you can combine policy type and in metric say if it was input or reference. I might not have full context of use-case so feel free to discard

Reply:
They refer to different things in the feature, my opinion is to keep it separate

Reply:
As for this we will only have Identity or Resource, so "Other" feels not so useful

Reply:
Got it